crates/bevy_animation/src/lib.rs

```diff
@@ -123,23 +123,50 @@ impl AnimationClip {
     }
 }
 
+/// Repetition behavior of an animation.
+#[derive(Reflect, Copy, Clone)]
+pub enum RepeatAnimation {
+    /// The animation will never finish.
+    Forever,
+    /// The animation will finish after running once.
+    Never,
```

Comment:
Maybe implement default with `Never`?

Reply:
Updated order and set default to be `RepeatAnimation::Never`.